src/create_shards.c

```diff
@@ -245,7 +240,7 @@ master_create_worker_shards(PG_FUNCTION_ARGS)
 		shardStorageType = SHARD_STORAGE_TABLE;
 	}
 
-	for (shardIndex = 0; shardIndex < shardCount; shardIndex++)
+	for (int64 shardIndex = 0; shardIndex < shardCount; shardIndex++)
```

Comment:
We generally don't declare variable on the loop itself. Should we move the declaration above?